fem/bilinearform_ext.cpp

```diff
@@ -125,6 +125,11 @@ void MFBilinearFormExtension::FormLinearSystem(const Array<int> &ess_tdof_list,
 
 void MFBilinearFormExtension::Mult(const Vector &x, Vector &y) const
 {
+
+   MFEM_VERIFY(interior_face_integs.Size() == 0,
+               "the case of interior face integrators is not"
+               " implemented");
```

Comment:
I think these are the extensions that may not support face integration, is this right @v-dobrev ?

Reply:
Actually, it looks like all `BilinearFormExtension` sub-classes support (at least some) face integrators. When I said that not all do, I did not actually check that.

Reply:
oh okay, I reverted the file.

Reply:
I've also tested this in my application and it the results look right.